guides/_subsections/configure-graphical-notifications.md

```diff
@@ -0,0 +1,27 @@
+## [Configure Graphical Notifications](#configure-graphical-notifications)
+Actions such as starting/stopping/archiving recordings and state changes such as running recordings stopping produce
```

Comment:
"running recordings stopping" refers to recordings that stop after a fixed duration, right? For clarity, what do you think of writing it as "recordings stopping after a fixed duration" to distinguish it from manual recording stops?

Reply:
That sounds good. Now it's making me wonder if we should be emitting a notification on manual stop, too.

Reply:
> Now it's making me wonder if we should be emitting a notification on manual stop, too.

Aren't we already emitting notifications on manual stop?

Reply:
I don't know. Probably? It sounds like something we would be doing... but I haven't checked :-)